protocol/contracts/libraries/Silo/LibLegacyTokenSilo.sol

```diff
@@ -0,0 +1,619 @@
+/**
+ * SPDX-License-Identifier: MIT
+ **/
+
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "../../C.sol";
+import "./LibSilo.sol";
+import "./LibUnripeSilo.sol";
+import "../LibAppStorage.sol";
+import {LibSafeMathSigned128} from "~/libraries/LibSafeMathSigned128.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/SafeCast.sol";
+import {LibBytes} from "~/libraries/LibBytes.sol";
+import "@openzeppelin/contracts/cryptography/MerkleProof.sol";
+
+/**
+ * @title LibLegacyTokenSilo
+ * @author Publius
+ * @notice Contains functions for depositing, withdrawing and claiming
+ * whitelisted Silo tokens.
+ *
+ * For functionality related to Seeds, Stalk, and Roots, see {LibSilo}.
+ */
+library LibLegacyTokenSilo {
+    using SafeMath for uint256;
+    using SafeCast for uint256;
+    using LibSafeMathSigned128 for int128;
+    using LibSafeMathSigned96 for int96;
+
+
+    //important to note that this event is only here for unit tests purposes of legacy code and to ensure unripe all works with new bdv system
```

Comment:
To my surprise I commented this event def out and it worked, then I saw it was only used when we were trying to support silo interactions without migration, since that's not supported it eliminated some tests and this function can be removed. https://github.com/BeanstalkFarms/Beanstalk/commit/3a2ec722fc63b84d01a8e0e27e27091553d917c8